uint8 READY = 0
uint8 RUNNING_BENCHMARK = 1

time timestamp
uint8 status
uint16 current_benchmark_seconds_left
string current_benchmark_info